src/DependencyInjection/CompilerPass/MakeCommandRegistrationPass.php

```diff
@@ -63,6 +63,14 @@ public function process(ContainerBuilder $container): void
                 }
             }
 
+            /*
+             * @deprecated remove this block when removing make:subscriber
+             */
+            if (method_exists($class, 'getCommandAlias')) {
+                $alias = $class::getCommandAlias();
+                $commandDefinition->addTag('console.command', ['command' => $alias, 'description' => 'Deprecated alias of "make:event"']);
+            }
```

Comment:
Should/could we leverage the `getCommandAliases()` referenced above since that logic already exists?

Reply:
+1, I found it a little bit weird to have methods like `getCommandAliases` whose existence leads to messages for very specific classes.

Maybe something like this?
```php
if (method_exists($class, 'getDeprecatedCommandAliases')) {
    foreach ($class::getDeprecatedCommandAliases() as $alias) {
        $commandDefinition->addTag('console.command', ['command' => $alias, 'description' => \sprintf('Deprecated alias of "%s"', $class::getCommandName())]);
    }
}
```